boards/shields/arduino_uno_click/boards/nrf9160dk_nrf9160.overlay

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2024 Golioth, Inc.
```

Comment:
Is Golioth copyright correct here ? (I mean are you doing this "for" Golioth ?).

Reply:
Yes, the copyright is correct (I do some contract work for Golioth).